components/director/Makefile

```diff
@@ -8,3 +8,15 @@ include $(SCRIPTS_DIR)/generic_make_go.mk
 VERIFY_IGNORE := /vendor\|/automock\|/testdata
 
 verify:: gqlgen check-gqlgen
+
+errcheck-local:
+	 GO111MODULE="on" errcheck -blank -asserts -ignoregenerated ./...
```

Comment:
Can we set GO111MODULe=on at the top on makefile and then we have to define it once.